x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceDetailTabs.tsx

```diff
@@ -52,7 +53,9 @@ export class ServiceDetailTabs extends React.Component<TabsProps> {
           defaultMessage: 'Metrics'
         }),
         path: `/${serviceName}/metrics`,
-        render: () => <ServiceMetrics urlParams={urlParams} />
+        render: () => (
+          <ServiceMetrics urlParams={urlParams} location={location} />
```

Comment:
`ServiceMetrics` was missing the `location` prop, so was using the global `location` implicitly.

Reply:
Nice catch, can we lint against using global `location`? Even if you want to use it you should use `window.location` ...

Reply:
There is a rule in ESLint [`no-implicit-global`](https://eslint.org/docs/rules/no-implicit-globals) but tslint doesn't have this :(